hack/make/build-deb

```diff
@@ -38,7 +38,7 @@ set -e
 	# TODO decide if it's worth getting go-md2man in _each_ builder environment to avoid this
 
 	# TODO add a configurable knob for _which_ debs to build so we don't have to modify the file or build all of them every time we need to test
```

Comment:
i was thinking something as simple as that, with the default in the absence of the env var being `*`